packaging/linux/arch/arch_common.sh

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env bash
+
+clone_maybe() {
+  if [ ! -e "$2" ] ; then
+    git clone "$1" "$2"
+  fi
+}
+
+setup_arch_build() {
+    build_root="$1"
+    git_url="$2"
+    src_prefix="$3"
+
+    # build_binaries.sh and deb/package_binaries.sh must already have
+    # been run on this build root.
+    here="$(dirname "$BASH_SOURCE")"
+
+    mkdir -p "$build_root/arch"
+
+    ###
+    ### keybase-bin
+    ###
```

Comment:
I think this section header comment used to pair with another one for the `keybase-git` section, but now that this is the only section probably we can delete it?